mcu/stm32f103.S

```diff
@@ -198,6 +198,15 @@ Reset_Handler:
     strb    r2, [r0, #RCC_APB2RSTR]
     strb    r2, [r0, #RCC_APB2ENR]
     cbz     r4, .L_start_boot
+
+#if (DFU_VERIFY_CHECKSUM != _DISABLE)
+    ldr     r0, =#_APP_START
+    ldr     r2, =#(__romend - 8)
```

Comment:
As per conversation, the adjustment should use a define from the checksum routine to determine the length, this would allow the checksum to be replaced with SHA or some other more advanced routine.

Reply:
Fixed https://github.com/dmitrystu/sboot_stm32/pull/10/commits/0b643c2f37857561fb61ae8bd86335b32ecb72a4